packages/frontend/src/components/Election/Voting/Thanks.tsx

```diff
@@ -41,16 +53,18 @@ const Thanks = () => {
                     </Typography>
                 }
 
-                <Box sx={{ width: '100%', display: 'flex', justifyContent: 'space-between', flexDirection: { xs: 'column', sm: 'row' } }} >
+                <Box sx={{ width: '100%', display: 'flex', justifyContent: 'space-between', flexDirection: { xs: 'column', sm: 'column' } }} >
+                    {election.state == 'open' && election.settings.ballot_updates &&
+                        MainButton({
```

Comment:
I think this should be a secondary button and probably rendered below the results button not on top (though they currently won't be shown at the same time)